index.html

```diff
@@ -0,0 +1,220 @@
+          <div class="program-item">
+            <span class="material-icons material-icons-outlined item-icon">
+              menu_book
+            </span>
+            <h3 class="item-title">Local missions</h3>
+            <p class="item-desc">The aim of this workshop is to show us how we can think about our areas of training
+              and
+              gifting as tools that
+
+            </p>
+          </div>
+          <div class="program-item">
+            <span class="material-icons material-icons-outlined item-icon">
+              laptop_chromebook
+            </span>
+            <h3 class="item-title">Technology Role</h3>
+            <p class="item-desc">The aim of this workshop is to show us how we can think about our areas of training
+              and
+              gifting as tools that
+
+            </p>
+          </div>
+          <div class="program-item">
+            <span class="material-icons material-icons-outlined item-icon">
+              watch
+            </span>
+            <h3 class="item-title">Urgency</h3>
+            <p class="item-desc">The aim of this workshop is to show us how we can think about our areas of training
+              and
+              gifting as tools that
+
+            </p>
+          </div>
+          <div class="program-item">
+            <span class="material-icons material-icons-outlined item-icon">
+              headset_off
+            </span>
+            <h3 class="item-title">Distractions</h3>
+            <p class="item-desc">The aim of this workshop is to show us how we can think about our areas of training
+              and
+              gifting as tools that
+
+            </p>
+          </div>
+        </div>
+        <div><a href="#" class="see-program">
+            <h4>SEE THE WHOLE PROGRAM</h4>
+          </a></div>
+        <div class="button-container"><button class="button" id="signup">Sign Up for Proclaim</button></div>
+      </section>
+      <!-- Featured speakers -->
+      <section class="featured">
+        <h2 class="program-heading">Featured Speakers</h2>
+        <hr class="line">
+        <div class="speakers"></div>
+         <div class="button-container">
+           <button>More<span class="material-icons material-icons-outlined arrow">
+               keyboard_arrow_down
+             </span></button>
+
+         </div>
+      </section>
+      <section class="partners partners-index" id="partner">
+        <h2 class="partner-title">partners</h2>
+        <hr class="line">
+        <ul class="partner-list">
+          <li class="partner">
+            <h2>Ekklesia</h2>
+          </li>
+          <li class="partner">
+            <h2>Biblica</h2>
+          </li>
+          <li class="partner">
+            <H2>Rbc</H2>
+          </li>
+          <li class="partner">
+            <h2>Hopecity</h2>
+          </li>
+          <li class="partner">
+            <h2>Mamlaka</h2>
+          </li>
+
+        </ul>
+      </section>
```

Comment:
- [ ] This partners section look all clumped together on mobile view(320px), please ensure there have the correct font size and are properly aligned.
![ice_screenshot_20210804-113718](https://user-images.githubusercontent.com/33323077/128168243-4d2fab55-29e2-4beb-9f5b-5f8b947df937.png)